packages/mobile/src/account/SyncBankAccountScreen.tsx

```diff
@@ -0,0 +1,72 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { StackScreenProps } from '@react-navigation/stack'
+
+import * as React from 'react'
+import { useAsync } from 'react-async-hook'
+import { useTranslation } from 'react-i18next'
+import { ActivityIndicator, View, Text, StyleSheet } from 'react-native'
+import { createFinclusiveBankAccount, exchangePlaidAccessToken } from 'src/in-house-liquidity'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import useSelector from 'src/redux/useSelector'
+import { mtwAddressSelector, walletAddressSelector } from 'src/web3/selectors'
+
+type Props = StackScreenProps<StackParamList, Screens.SyncBankAccountScreen>
+
+const SyncBankAccountScreen = ({ route }: Props) => {
+  const { t } = useTranslation()
+  const accountMTWAddress = useSelector(mtwAddressSelector) || ''
+  const walletAddress = useSelector(walletAddressSelector) || ''
+  const { publicToken } = route.params
+
+  useAsync(async () => {
+    const accessTokenResponse = await exchangePlaidAccessToken({
+      accountMTWAddress,
+      walletAddress,
+      publicToken,
+    })
+    if (!accessTokenResponse.ok) {
+      // TODO(wallet#1447): handle errors from IHL
+      return
+    }
+    const { accessToken } = await accessTokenResponse.json()
+
+    const finclusiveBankAccountResponse = await createFinclusiveBankAccount({
+      accountMTWAddress,
+      walletAddress,
+      plaidAccessToken: accessToken,
+    })
+    if (!finclusiveBankAccountResponse.ok) {
+      // TODO(wallet#1447): handle errors from IHL
+      return
+    }
+    // TODO(wallet#1449): redirect to Bank Account List Page
+  }, [])
+
+  return (
+    <View style={styles.container}>
+      <ActivityIndicator size="small" color={colors.greenBrand} />
+      <Text style={styles.connecting}>{t('syncingBankAccount')}</Text>
+    </View>
+  )
+}
+
+SyncBankAccountScreen.navigationOptions = () => {
+  return {}
```

Comment:
do we need to set navigationOptions here if it returns {} ?

Reply:
It looks like if we just dont want any header to be shown, there's an option called `noHeader` in `StackNavigationOptions`